diff -up libspnav-1.1/Makefile.in.omv~ libspnav-1.1/Makefile.in
--- libspnav-1.1/Makefile.in.omv~	2024-05-05 23:00:58.338924179 +0200
+++ libspnav-1.1/Makefile.in	2024-05-05 23:01:18.107105052 +0200
@@ -11,7 +11,7 @@ libpaths = -L/usr/local/lib -L/usr/X11R6
 CC ?= gcc
 AR ?= ar
 CFLAGS = $(opt) $(dbg) -std=c89 $(pic) -pedantic -Wall -fno-strict-aliasing $(incpaths) $(user_cflags)
-LDFLAGS = $(libpaths) $(user_ldflags) $(xlib)
+LDFLAGS = $(libpaths) $(user_ldflags) $(xlib) -lm
 
 ifeq ($(shell uname -s), Darwin)
 	lib_so = libspnav.dylib
